elife/scripts/rds-perms2.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
please rename the suffix of this file to something more expressive than `2`

Reply:
done